src/cloudsc_fortran_pyiface/pythonsrc/dwarfdriver.py

```diff
@@ -0,0 +1,83 @@
+"""
+Driver that:
+- loads input variables and parameters from .h5 file,
+- invokes Fortran kernel computation,
+- validates against reference results read from another .h5 file.
+"""
+import sys
+from pathlib import Path
+from operator import itemgetter
+import cloudsc as clsc
+from cloudsc_data import define_fortran_fields
+from cloudsc_data import load_input_fortran_fields, load_input_parameters
+from cloudsc_data import load_reference_fields
+from cloudsc_data import convert_fortran_output_to_python
+from cloudsc_data import cloudsc_validate
+sys.path.append('../../build/src/cloudsc_fortran_pyiface')
```

Comment:
Since you're already setting relative paths here, I'd move the import of `cloudsc as clsc` further down and possibly force the load-path detection here. Something like (untested):
```
from importlib import import_module

here = Path(__file)
cldir = here / '<relative path to cloudsc parent directory>'

if cldir not in sys.path:
    sys.path.append(cldir)

cloudsc = import_module('cloudsc')